docs/Learning-Environment-Examples.md

```diff
@@ -157,13 +157,13 @@ If you would like to contribute environments, please see our
 ![Reacher](images/reacher.png)
 
 * Set-up: Double-jointed arm which can move to target locations.
-* Goal: The agents must move it's hand to the goal location, and keep it there.
-* Agents: The environment contains 10 agent linked to a single brain.
+* Goal: The Agents must move it's hand to the goal location, and keep it there.
+* Agents: The environment contains 10 Agent linked to a single Brain.
 * Agent Reward Function (independent):
-  * +0.1 Each step agent's hand is in goal location.
-* Brains: One brain with the following observation/action space.
+  * +0.1 Each step Agent's hand is in goal location.
+* Brains: One Brain with the following observation/action space.
   * Vector Observation space: 26 variables corresponding to position, rotation,
-    velocity, and angular velocities of the two arm Rigidbodies.
+    velocity, and angular velocities of the two arm Rigid Bodies.
```

Comment:
@vincentpierre I think Vince mentioned that Rigidbodies should be one word since it's a Unity thing? https://docs.unity3d.com/ScriptReference/Rigidbody.html